src/StaticCaching/Cachers/AbstractCacher.php

```diff
@@ -294,4 +296,19 @@ protected function getPathAndDomain($url)
             $parsed['scheme'].'://'.$parsed['host'],
         ];
     }
+
+    public function warmUrl(string $url): void
+    {
+        // TODO: use same parameters as the StaticWarm command?
+        $client = new Client();
```

Comment:
In the `StaticWarm` command, two parameters get passed into the Guzzle client - one for SSL verify & the other for HTTP Auth.

What should we do about those parameters in this case, where we can't get the parameters from the user when they're running the command?

https://github.com/statamic/cms/blob/f8fcb062488507c2dc96e96b8a2377ce8c5c7246/src/Console/Commands/StaticWarm.php#L74-L79

Reply:
I've implemented a check for SSL verification when running locally but not sure what to do about the other cases.